src/modules/Root/Root.jsx

```diff
@@ -10,27 +10,34 @@ import Header from '../Layout/Header/Header';
 import Footer from '../Layout/Footer/Footer';
 import APIRespondModalContainer from '../modals/APIRespondModal/apiRespondModalContainer';
 
+import LoginPage from '../pages/LoginPage/LoginPage';
 import ApiListPage from '../pages/ApiListPage/ApiListPage';
 import NewApiPage from '../pages/NewApiPage/NewApiPage';
 import EditApiPage from '../pages/EditPage/EditApiPage';
 
+import block from '../../helpers/bem-cn';
+
+const nav = block('j-header__nav-item');
+
 class Root extends Component {
   render() {
     return (
       <Router>
-        <div className="App">
+        <div className="j-app">
           <Header>
-            <NavLink exact to="/" className="j-nav__item" activeClassName="j-nav__item--active">API Definitions</NavLink>
-            <NavLink to="/new" className="j-nav__item" activeClassName="j-nav__item--active">New API</NavLink>
+            <NavLink exact to="/" className={nav()} activeClassName={nav({active: true})()}>API Definitions</NavLink>
```

Comment:
Actually, maybe it's better to make this fix also in separate **PR**, as far it's related not only for this task, how do you think, @nik-kor ?